docs/en/query_language/functions/hash_functions.md

```diff
@@ -66,50 +66,50 @@ Levels are the same as in URLHierarchy. This function is specific to Yandex.Metr
 
 ## farmHash64
 
-Calculates SipHash from a string.
+Calculates FarmHash64 from a string.
 Accepts a String-type argument. Returns UInt64.
-For more information, see the link: <https://github.com/google/farmhash>
+For more information, see the link: [FarmHash64]<https://github.com/google/farmhash>
```

Comment:
round brackets: `[FarmHash64](https://github.com/google/farmhash)`
here and in other similar places